connector.go

```diff
@@ -39,10 +38,10 @@ func (c *connector) Deploy(ctx context.Context, image string) (deployer.Plugin,
 		return nil, err
 	}
 
-	//nolint:godox
-	// TODO: Make this sleep workaround no longer needed.
 	// It's required to not crash podman
-	time.Sleep(500 * time.Millisecond)
+	if c.config.Deployment.StartDelayMS > 0 {
+		time.Sleep(time.Duration(c.config.Deployment.StartDelayMS) * time.Millisecond)
+	}
```

Comment:
You could have avoided the second fetch with a local, but that's an even smaller impact on time/CPU so I won't even comment on that. (Oops.)

Reply:
Hopefully Go has an common-expressions optimizer which will take care of that.  🙂